### DISCUSSION TITLE
Using local storage for oVirt Engine?

### DISCUSSION THREAD

**Discussion by smittyinthesky** [oVirt/ovirt-site#3039] [Q&A]
I believe the only way to use local storage for ovirt-engine is to export the local filesystem over NFS. Are there any plans to support a less awkward local storage solution?

**Comment by frangdlt** [comment]
What is your use case here? Are you trying to test the deployment of Self-hosted engine? Or trying to build a Manager for production usage?

If you are doing the latter, you can deploy ovirt-engine (Manager) in a dedicated server/VM outside the hypervisors, using local storage. You will not have High Availability tough, as you are using local storage. The system chosen to run the ovirt-engine daemon cannot be a hypervisor.
